product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,265 @@
+
+If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing_pgd_for_edb_postgres_extended_server) and [Installing PGD for Community Postgresql](#installing_pgd_for_community_postgresql).
+
+## Worked example
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+For example, if we are installing EDB Postgres Advanced Server (epas) version 16, we would install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: `edb-pgd5-proxy` for PGD Proxy and `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Initializing and starting the database
+
+Now initialise the database. You will need to know the path to the setup script for your particular Postgres. 
+For EDB Postgres Advanced Server, this script can be found in `/usr/edb/as16/bin` as `edb-as-16-setup`. 
+This command needs to be run with the `initdb` parameter and we need to pass an option setting the database to use UTF-8.
+
+
+```
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+```
+
+Once the database is initialised, we will start it which will enable us to continue configuring the BDR extension.
+
+```
+sudo systemctl start edb-as-16
+```
+
+### Configuring the BDR extension
+
+Installing EDB Postgres Advanced Server creates a system user `enterprisedb` with admin capabilities when connected to the database.
+
+#### Preload the BDR library
+
+We want the bdr library to be preloaded with other libraries. 
+EPAS has a number of libraries already preloaded, so we have to prefix the existing list the the BDR library.
+
+```
+echo -e "shared_preload_libraries = '\$libdir/bdr,\$libdir/dbms_pipe,\$libdir/edb_gen,\$libdir/dbms_aq'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+```
+
+!!!tip
+This command format (`echo ... | sudo ... tee ...`) appends the echoed string to the end of the postgresql.conf file which is owned by another user.
+!!!
+
+#### Set the `wal_level`
+
+The BDR extension needs to set the server to perform logical replication. We do this by settin `wal_level` to `logical`.
+
+```
+echo -e "wal_level = 'logical'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+```
+
+#### Enable commit timestamp tracking
+
+The BDR extension also needs the commit timestamp tracking enabled.
+
+```
+echo -e "track_commit_timestamp = 'on'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+```
+
+#### Add a password to the Postgres enterprisedb user
```

Comment:
I think it'd be more consistent to put this step after `Raise max_worker_processes` so all the GUC changes are together; this also matches the ordering in the respective summaries below.

Reply:
Ah, missed moving that up. Thanks. Fixed.